### DISCUSSION TITLE
vorrei creare pagina "personalizzazioni" per .......

### DISCUSSION THREAD

**Discussion by cirospat** [opendatasicilia/ods-mkdocs-material#29] [Idee/Proposte]
.... per dare le informazioni agli utenti su cosa devono - come minimo - cambiare nelle impostazioni del progetto clonato, al fine di avere la loro documentazione online correttamente esposta.
Una pagina in cui facciamo l'elenco delle cose minime da personalizzare.

Esempio:

- i parametri del file `mkdocs.yml` per dare titolo al progetto;
- i contenuti nel file `index.md` che si trova nella directory `docs`;
- i contenuti dei file `.md` che si trovano dentro la cartella madre `docs` (ed eventualmente sotto cartelle) e che rappresentano il nome dei capitoli o sezioni del documento. Ogni file `.md` è la pagina web del progetto di documentazione;
- inserire le immagini nella cartella `img` che si trova nel percorso `mkdocs-style/docs/img/` per esporre l'icona e il favicon del progetto di documentazione. E' facoltativo e non è importante;
- inserire i dati nel file `docs/manifest.webmanifest`;
- eventuali altre cose da fare per personalizzare il progetto di documentazione.

Che ne pensate?
Servirà come elenco delle cose minime da fare obbligatoriamente.

**Comment by aborruso** [comment]
Caro @cirospat , mi trovi d'accordo e per me va tutto nella parte di installazione.

Alcuni dei punti che hai citato non li metterei lì, ma in altre sezioni. Ma poi ne parliamo e decidiamo.

Mi trovi d'accordo

P.S. ho spostato questa issue, nelle discussions, perché per ora è un'idea, una proposta. Poi nelle issue mettiamo le cose che cambieranno i contenuti e il codice

**Comment by cirospat** [comment]
Ok


<scusa la brevità, rispondo da mobile>

Il mar 26 ott 2021, 13:08 Andrea Borruso ***@***.***> ha
scritto:
